subworkflows/local/reference_indexing_single.nf

```diff
@@ -109,7 +112,8 @@ workflow REFERENCE_INDEXING_SINGLE {
                                 }
 
     emit:
-    reference            = ch_ref_index_single.reference             // [ meta, fasta, fai, dict, mapindex, circular_target ]
+    reference            = ch_ref_index_single.reference             // [ meta, fasta, fai, dict, mapindex ]
+    circularmapper       = ch_ref_index_single.circularmapper        // [ meta, circular_target, circularmapper_elongated_fasta, circularmapper_elongated_fai ]
```

Comment:
```suggestion
    elongated_reference       = ch_ref_index_single.circularmapper        // [ meta, circular_target, circularmapper_elongated_fasta, circularmapper_elongated_fai ]
```

Reply:
These need changing also where the output is picked up in the `REFERENCE_INDEXING` SWF